content/en/monitors/types/ci.md

```diff
@@ -20,22 +20,28 @@ further_reading:
 ---
 
 {{< site-region region="gov" >}}
-<div class="alert alert-warning">CI Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+<div class="alert alert-warning">This feature is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
 {{< /site-region >}}
 
 ## Overview
 
-Once [CI Visibility is enabled][1] for your organization, you can create a CI Pipeline or CI Test monitor.
+To create a monitor for CI pipelines, CI tests, or CD deployments, first enable the related product for your organization:
 
-CI monitors allow you to visualize CI data and set up alerts on it. For example, create a CI Pipeline monitor to receive alerts on a failed pipeline or a job. Create a CI Test monitor to receive alerts on failed or slow tests.
+| Monitor type     | Required product         |
+|------------------|--------------------------|
+| CI Pipeline      | [CI Visibility][1]       |
+| CI Test          | [Test Optimization][2]   |
+| CD Deployments   | [CD Visibility][3]       |
+
+CI/CD and test monitors allow you to visualize CI/CD data and set up alerts on it. For example, create a CI Pipeline monitor to receive alerts on a failed pipeline or a job. Create a CI Test monitor to receive alerts on failed or slow tests.
```

Comment:
```suggestion
CI/CD and Test monitors allow you to visualize CI/CD data and set up alerts on it. For example, create a CI Pipeline monitor to receive alerts on a failed pipeline or a job. Create a CI Test monitor to receive alerts on failed or slow tests.
```
I think this should be capitalized throughout? As with it lowercase, it read like it was "testing" something vs the name of the monitor.